api/client/utils.go

```diff
@@ -343,3 +357,13 @@ func readBody(stream io.ReadCloser, hdr http.Header, statusCode int, err error)
 	}
 	return body, statusCode, nil
 }
+
+// serverOSFromServerHeader parses the HTTP header returned from the docker
+// daemon to extract the OS of the server. Header format is Docker/Version (OS)
+func serverOSFromServerHeader(serverHeader string) string {
```

Comment:
Perhaps this func should return a struct with all of the bits split out instead of just the OS.
```
struct {
  app string  // Docker
  version string   // 1.7.0
  os string  // windows
}
```
then we don't need one func per field.